### ISSUE TITLE
Default port?

### ISSUE THREAD

**Issue by djmadeira** [sindresorhus/get-port#6]
Any interest in accepting a PR to support passing in a preferred/default port?

**Comment by sindresorhus** [comment]
What's the point on having a preferred port if it's not guaranteed? I'm interested in your use-case.

**Comment by dylang** [comment]
As a developer working on a web server based project, like something that uses Express or Koa, it's 
nice to have a predictable port when running `yarn start`. 

* Chrome's autocomplete is port-particular
* If have a mechanism to automatically open in the browser and re-use the same tab then the port must be the same.
* Chrome can autocomplete the path getting you into a page you were working on earlier.

However, you can't hardcode that port, because what if another project is using it at the moment, or you have another instance running in another tab. Then it's nice that it will choose a different number.

For example, we do all of our local development on `localhost:3000`, but if we run two projects at the same time, the second project may need to use `localhost:3001`. This also informs the developer: "hey, did you know you were already running something on 3000?" without needing a warning message, because sometimes you just forget to shut down a process.

TLDR:
* Predictable pattern for development.
* Better than just failing when the port is already used.
